Modules/CustomRoleSelector.cs

```diff
@@ -95,7 +95,10 @@ or CustomRoles.TimeMaster
                     or CustomRoles.Lighter
                     or CustomRoles.Chameleon
                     or CustomRoles.Mario // Vector
-                    or CustomRoles.Wraith) continue;
+                    or CustomRoles.Wraith
+                    or CustomRoles.Arsonist) continue;
```

Comment:
Arsonist was been fixed on this map so he can appear on this map

Reply:
wdym? arsonist needs to vent after dousing everyone, right?

Reply:
Yes, but only once, when all players are doused

Reply:
so how is arsonist going to vent?
Also what about the setting where it can vent to kill set amount of players?

Reply:
Vents is only partially broken, but not completely

Reply:
> so how is arsonist going to vent? Also what about the setting where it can vent to kill set amount of players?

Damn, I forgot he can do that :/

Reply:
Fix it later